routing/routing.xml

```diff
@@ -723,90 +723,157 @@
 		</way>
 
 		<way attribute="obstacle_srtm_alt_speed">
+
 			<if param="height_obstacles">
+
 				<gt value1="-100" value2=":incline">
-					<select value="-1" t="surface" v="cobblestone"/>
-					<select value="-1" t="osmand_highway_integrity_brouting_low" v="yes"/>
-					<select value="-1" t="highway" v="path"/>
-					<select value="-1" t="highway" v="track"/>
-					<select value="-1" t="highway" v="footway"/>
+                	                <select value="-1"/>
```

Comment:
We shouldn't completely forbid slopes < -100 because insufficient elevation data accuracy can produce even higher values at some hilly areas. These roads can be the only way to get to destination.